server/api/audit_boards.py

```diff
@@ -59,25 +59,30 @@ def assign_sampled_ballots(
         .filter_by(jurisdiction_id=jurisdiction.id)
         .join(SampledBallot.draws)
         .filter_by(round_id=round.id)
-        .order_by(Batch.name)  # group_by prefers a sorted list
         .options(contains_eager(SampledBallot.batch))
```

Comment:
should this be ordered by a sampled ballot ID or something else, just to make sure there is a consistent ordering whenever you query this more than once?

Reply:
hmm yeah ill figure something out

Reply:
actually the order doesn't matter because the next thing that happens is it gets run through `group_by`